.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: swift
+os: osx
+osx_image: xcode9.2
+xcode_project: JOSESwift.xcodeproj
+xcode_workspace: JOSESwift.xcworkspace
+xcode_scheme: Tests
+
+script:
+  - bundle exec fastlane scan -s JOSESwift
```

Comment:
does `bundle install` happen automatically?

Reply:
jep 👍

Reply:
cool how?

Reply:
I guess travis searches for the `Gemfile` 🤔